Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM ubuntu:bionic
```

Comment:
I don't know enough about `Dockerfile`s to properly review this, but I guess if it runs the :+1:.